### ISSUE TITLE
Matching incoming call to core account

### ISSUE THREAD

**Issue by NLLAPPS** [BelledonneCommunications/linphone-android#1543]
Hi, I have been playing around with linphone android app and sdk. I can see that you are creating one phone account according to default account on Core ( https://github.com/BelledonneCommunications/linphone-android/blob/19b15e2b3b91e3d355d18db88663cd1d47c65cee/app/src/main/java/org/linphone/telecom/TelecomHelper.kt#L140 )

But that may not be suitable when integrated with Android Telecom as user would need to keep chancing default Core Account when making outgoing calls with different Core Accounts as Telecom would see one account.

In order to avoid this you may need to create multiple Telecom Accounts as user adds new Core Account (SIP account) to your app. 

However, there isn't seem to be a way matching incoming SIP call to an existing account available in Core. 

So, this is the main issue I hit upon when I made required changes to create multiple Telecom Accounts per Core Account.

Outgoing calls can be matched by using account.params.identityAddress.asStringUriOnly() as Telecom PhoneAccounHandle ID or address.

But, when there is an incoming call and Core Call.State.IncomingReceived  called, you only get Call object and it does not have any information about which Core Account or AuthInfo this call belongs to.

You cannot derive this information from call.callLog.toAddress or call.callLog.localAddress 

 toAddress can be anything (imagine call forwarding) 

localAddress seems be local IP at times (when SIP server is entered as IP) 

or username part of localAdress seems to be changing ( for example: I teste with sipgate and saw that call.callLog.localAddress.asStringUriOnly() for account sip:123456@sipgate.co.uk comes as 123456**e0**@sipgate.co.uk **e0** added to the username 

**So basically, what I am trying to ask is this. How can I identify which account the incoming call is coming to?**

**Comment by Viish** [comment]
Duplicate of https://github.com/BelledonneCommunications/liblinphone/issues/195
